lapis/src/main/kotlin/org/genspectrum/lapis/openApi/OpenApiDocs.kt

```diff
@@ -423,6 +462,16 @@ private fun detailsMetadataFieldSchemas(databaseConfig: DatabaseConfig) =
         .metadata
         .associate { it.name to Schema<String>().types(setOf(mapToOpenApiType(it.type))) }
 
+private fun mostRecentCommonAncestorMetadataFieldSchemas(): Map<String, Schema<*>> =
+    mapOf(
+        "mrcaNode" to StringSchema()
+            .description("The label of the most recent common ancestor node in the phylogenetic tree."),
+        "missingNodeCount" to IntegerSchema()
+            .description("The number of nodes in the filter not in the phylogenetic tree."),
+        "missingFromTree" to StringSchema()
+            .description("The comma-separated list of names of the nodes in the filter not in the phylogenetic tree."),
```

Comment:
If we already know that this is a list - why don't we return a list then? Is there a good reason to return a csv string instead?

Reply:
Why does SILO return a string then when it already knows that it's a list?

Reply:
@Taepper ah so SILO doesn't currently support nested types? 

Then lets keep it like this for now :-)

Reply:
Do you think it makes sense to do the mapping in LAPIS then for now (csv string -> list)? (Not necessarily in this PR though)

Reply:
So tbh I think its quite unlikely that there will be any nodes in this field (unless our tree is very out of sync with our data - and I think we should anyways try to avoid that) so I'm not sure it makes much sense to format it much

Reply:
Will it be used? If yes, then I think we should provide a proper API, especially since it seems very easy in this case. If no, why do we have it at all?

Reply:
Then let's create an issue so that we don't forget about it?

Reply:
https://github.com/GenSpectrum/LAPIS/issues/1285